src/main/java/io/cdap/e2e/pages/actions/CdfConnectionActions.java

```diff
@@ -129,7 +129,7 @@ private static void selectConnectionDataRow(String dataRow) {
     WaitHelper.waitForElementToBeClickable(CdfConnectionLocators.searchDirectoryInput,
                                            ConstantsUtil.DEFAULT_TIMEOUT_SECONDS);
     ElementHelper.sendKeys(CdfConnectionLocators.searchDirectoryInput, dataRow);
-    WaitHelper.waitForTextToBePresentInElementValue(CdfConnectionLocators.searchDirectoryInput, dataRow);
+    WaitHelper.waitForPageToLoad();
```

Comment:
should we do both?

```
WaitHelper.waitForPageToLoad();
WaitHelper.waitForTextToBePresentInElementValue(CdfConnectionLocators.searchDirectoryInput, dataRow);
```
@poojantcs wdyt?

Reply:
Yes, we should.

Reply:
We can use both the steps but right now we are facing error on that particular step in which this step is waiting for tableName to be present in search bar. But it was not showing. So we just remove this step and simply load that page.
![embedding_-643155356](https://github.com/user-attachments/assets/cfbbf89d-25a9-4e80-980c-1bb28e6968ba)
As in the above image the tableName was searched but it is not showing in search bar so we did this change.

Reply:
so looks like we are not correctly sending the input string in search bar?

Because I can see it in search bar when I do it manually?

![image](https://github.com/user-attachments/assets/82426e46-43ef-4141-851a-777ea7e11567)

Reply:
No, we are sending right input in search bar because the table only filters out when we search that table name in search bar. And in the above error image we can see the table name with UUID comes out. And this step is only failing on github actions neither on CDF nor when we do it locally.